docs/docs/Components/components-vector-stores.md

```diff
@@ -78,6 +80,54 @@ For an example of using the **Astra DB Vector Store** component with an embeddin
 
 For more information, see the [Astra DB Serverless documentation](https://docs.datastax.com/en/astra-db-serverless/databases/embedding-generation.html).
 
+### Hybrid search
+
+The **Astra DB** component includes **hybrid search** enabled by default.
+
+The component fields related to hybrid search are **Search Query**, **Lexical Terms**, and **Reranker**.
+
+* **Search Query** finds results by vector similarity.
+* **Lexical Terms** is a comma-separated list of keywords.
+* **Reranker** is the re-ranker model used in the hybrid search.
+The default model is `nvidia/llama-3.2-nv.reranker`.
+
+**Hybrid search** passes a vector similarity search and a lexical search to the [find and rerank Data API endpoint](https://docs.datastax.com/en/astra-db-serverless/api-reference/document-methods/find-and-rerank.html), where a reranker combines and sorts the results based on the vector search's similarity to the lexical terms.
+
+To use **Hybrid search** in the **Astra DB** component, do the following:
+
+1. To create a **Hybrid Search** template flow, click **New Flow** > **RAG** > **Hybrid Search**.
+2. In the **OpenAI** model component, add your **OpenAI API key**.
+3. In the **Astra DB** vector store component, add your **Astra DB Application Token**.
+4. In the **Database** field, select your database.
+5. In the **Collection** field, select the collection you want to search.
```

Comment:
The collection has to be configured for hybrid search at the time that you create the collection. A preexisting collection wont work.

Also, the documents in the collection must have the necessary values to work with hybrid search.

Is langflow/open AI handling this for the user somehow?